projects/sandbox/src/app/dataset-content-summary/dataset-content-summary.component.html

```diff
@@ -117,8 +117,12 @@
   </span>
 
   <span class="grid-header search-header">
-    <span class="search-icon"></span>
-    <input placeholder="search {{ gridData.length }} record ids" />
+    <span class="search-icon" [ngClass]="{ active: filterTerm.length > 0 }"></span>
+    <input
+      placeholder="search {{ gridData.length }} record ids"
+      [(ngModel)]="filterTerm"
+      (keyup)="updateTerm($event)"
+    />
   </span>
 
```

Comment:
I see this link in the grid is not implemented (line 167), just displays the record to be a clickable link but no action occurs.
```
<a class="field-label-mobile" (click)="sortHeaderClick()">Record Id</a
        >
```